website/content/docs/concepts/topologies.md

```diff
@@ -2,6 +2,15 @@
 title: Heron Topologies
 ---
 
+> ## New DSL for Heron
+> As of Heron version 0.15.2, there is a new **Heron DSL** that can be used
```

Comment:
This should be 0.16.0

Reply:
Please note that this is a work in progress